source/_patterns/typography/examples.hbs

```diff
@@ -21,9 +21,27 @@
 			<h6 data-variant="light">Headline 6</h6>
 		</db-tab>
 		<db-tab label="Body" name="tab-bar">
+			<h1>functional</h1>
 			<p>Primary font: "DB Screen Sans", "Helvetica", "Arial", sans-serif;</p>
 			<p><em>Primary font italic: "DB Screen Sans", "Helvetica", "Arial", sans-serif;</em></p>
 			<p><strong>Primary font bold: "DB Screen Sans", "Helvetica", "Arial", sans-serif;</strong></p>
+			<p class="DO-NOT-COPY-THIS-CLASS-example-typography-tiny">I'm Tiny Tina</p>
+
+			<div class="db-ui-regular">
```

Comment:
I would expect `class="db-ui-regular"` to be `data-tonality="regular"`, adapted by the other attributes like e.g. size, variant, etc.

Reply:
Add data-tonality as an attribute as well.

And I created a new issue https://github.com/db-ui/base/issues/175 for the colors. Let's discuss it here for the colors because this could be tricky. We use data-color=XXX for some components, so we might find another name for the base-colors?

For the scaling and typography, we removed the classes for now because with these changes we have the css-properties inside them. So there is no need for any class or data-attribute, it will be set by tonality and we want to restrict it. Otherwise, there would be a mismatch with fonts and scaling.